### ISSUE TITLE
Error when launching the game with endless mode

### ISSUE THREAD

**Issue by Dhruv00000** [acmol/HarderKCM#4]
When I launch the game with the endless mode mod, it shows this error-:

```
[Warning:  HarmonyX] AccessTools.DeclaredMethod: Could not find method for type DiskCardGame.CopyCardNodeData and name GenerationPrerequisiteConditions and parameters
[Error  : Unity Log] ArgumentException: Undefined target method for patch method static void HarderKCM.CopyCardNodeData_GenerationPrerequisiteConditions::Postfix(System.Collections.Generic.List`1[[DiskCardGame.NodeData+SelectionCondition, Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null]]& __result)
Stack trace:
HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
Rethrow as HarmonyException: Patching exception in method null
HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.<PatchAll>b__11_0 (System.Type type) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.PatchAll (System.Reflection.Assembly assembly) (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarmonyLib.Harmony.PatchAll () (at <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0)
HarderKCM.API.Awake () (at <fa4fc1b09489451ca76c917661854d23>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.Application:.cctor()
Sirenix.Serialization.UnitySerializationInitializer:Initialize() (at X:/Repositories/sirenix-development/OdinSerializer/OdinSerializer/Unity Integration/UnitySerializationInitializer.cs:53)
Sirenix.Serialization.UnitySerializationInitializer:InitializeRuntime() (at X:/Repositories/sirenix-development/OdinSerializer/OdinSerializer/Unity Integration/UnitySerializationInitializer.cs:77)
```

**Comment by acmol** [comment]
This mod only works with Kaycee's mod. 
I guess your game is not with Kaycee's mod?

**Comment by acmol** [comment]
Err... I repeated this bug. I will fix it  soon. Thank you for reporting it.

**Comment by acmol** [comment]
Fixed in version 0.0.8

**Comment by Dhruv00000** [comment]
> Fixed in version 0.0.8

Thank you, btw if I do a non-kc run, will it still make it infinite or will the run only be infinite if I do a kc run
